.eslintrc

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "@antfu"
```

Comment:
this should be adapted, not removed maybe check https://github.com/antfu/vitesse-nuxt3

Reply:
Vitesse was the base i used for both websites

Reply:
Vite is supported by default in Nuxt 3, the Antfu's template just bring UnoCSS support